frontend/src/app/pages/marker-tutorial-page/marker-tutorial-page.component.ts

```diff
@@ -75,4 +83,14 @@ export class MarkerTutorialPageComponent implements OnInit {
         this.fourthStepVideo.nativeElement.play();
     }
 
+    public endTutorial(): void {
+        this.user.settings.skipTutorial = true;
+        sessionStorage.setItem('userInfo', JSON.stringify(this.user));
+
+        let settings: UserSettings = new UserSettings();
+        settings.skipTutorial = true;
```

Comment:
Please apply this value **after** you make sure that this field was saved on the backend side (next to line 93) :)